presto-hive/src/main/java/io/prestosql/plugin/hive/HiveConnectorFactory.java

```diff
@@ -51,6 +56,21 @@ public ConnectorHandleResolver getHandleResolver()
     @Override
     public Connector create(String catalogName, Map<String, String> config, ConnectorContext context)
     {
-        return createConnector(catalogName, config, context, Optional.empty());
+        ClassLoader classLoader = context.duplicatePluginClassLoader();
+        try {
+            Class<?> hiveConnectorFactoryClass = classLoader.loadClass(HIVE_CONNECTOR_FACTORY);
+            checkState(hiveConnectorFactoryClass != null, "Could not load %s from duplicate plugin class loader", HIVE_CONNECTOR_FACTORY);
```

Comment:
Use `verify()`

Reply:
`classLoader.loadClass` is not allowed to return null, so yes, `verify` is more appropriate

Reply:
The original intent was to verify that the class was actually loaded (we assume it meets the API contract -- if not the next line will NPE and that is fine). So we can remove it.